CHANGES.md

```diff
@@ -1,57 +1,30 @@
 # Change Log
 
-## Unreleased
```

Comment:
Add a new "Unreleased" block with empty changelog

Reply:
Going to do a commit after release adding this ...